test/DynamoCoreWpfTests/WorkspaceSaving.cs

```diff
@@ -507,7 +517,7 @@ public void HomeWorkspaceHasUnsavedChangesPropertyIsSetOnSaveAs()
         }
 
         // TODO: Enable when Open() is expanded to open Json
-        [Test, Ignore]
+        [Test]
```

Comment:
Is this a result of this change or even without?

Reply:
without, we have open now working so this test works now.